packages/SqueakMaps-APIs.package/SMAORSRoutingAPI.class/instance/getBoundingBoxFrom..st

```diff
@@ -4,4 +4,4 @@ getBoundingBoxFrom: aJson
 	| coordinates |
 	
 	coordinates := aJson at: 'bbox'.
-	^ { (coordinates first) @ (coordinates second) . (coordinates third) @ (coordinates fourth)}
\ No newline at end of file
+	^ coordinates first @ coordinates second corner: coordinates third @ coordinates fourth
```

Comment:
Hier vielleicht auch nochmal auslagern in `lowerLeftCorner` und `upperRightCorner`

Reply:
Gefixt in f717bcfcade633e3ad511dcaf4786d2d61ec4c68